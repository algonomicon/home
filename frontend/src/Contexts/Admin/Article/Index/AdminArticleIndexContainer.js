const AdminArticleIndexContainer = () => (
  <div>Article Index</div>
)

export default AdminArticleIndexContainer