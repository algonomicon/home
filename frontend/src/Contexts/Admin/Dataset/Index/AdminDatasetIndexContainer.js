import { inject, observer } from 'inferno-mobx'
import { Link } from 'inferno-router'

const AdminDatasetIndexContainer = ({ DatasetStore }) => {
  DatasetStore.fetchDatasets()

  return (
    <div>
      { DatasetStore.datasets.map(dataset => (
        <div>
          <Link to={`/admin/datasets/${dataset.slug}`}>{dataset.name}</Link>
        </div>
      ))}
    </div>
  )
}

export default inject('DatasetStore')(observer(AdminDatasetIndexContainer))