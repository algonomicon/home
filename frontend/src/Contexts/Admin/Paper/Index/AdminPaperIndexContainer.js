const AdminPaperIndexContainer = () => (
  <div>Paper Index</div>
)

export default AdminPaperIndexContainer