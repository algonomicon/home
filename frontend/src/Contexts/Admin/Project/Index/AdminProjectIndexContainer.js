const AdminProjectIndexContainer = () => (
  <div>Project Index</div>
)

export default AdminProjectIndexContainer