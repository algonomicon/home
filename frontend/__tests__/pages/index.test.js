import React from "react"
import renderer from "react-test-renderer"

import Index from "../../src/pages/index"

describe("Index", () => {
  it("renders correctly", () => {
    const data = {
      "file": {
        "childImageSharp": {
          "fluid": {
            "base64": "data:image/jpeg;base64,/9j/2wBDABALDA4MChAODQ4SERATGCgaGBYWGDEjJR0oOjM9PDkzODdASFxOQERXRTc4UG1RV19iZ2hnPk1xeXBkeFxlZ2P/2wBDARESEhgVGC8aGi9jQjhCY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2P/wgARCAAbABQDASIAAhEBAxEB/8QAGAAAAwEBAAAAAAAAAAAAAAAAAAMEAgX/xAAWAQEBAQAAAAAAAAAAAAAAAAABAAL/2gAMAwEAAhADEAAAAVtgC6pyyqMUKNYGif/EAB4QAAIBBAMBAAAAAAAAAAAAAAABAgQRITEDEhNC/9oACAEBAAEFAvfkIVLUpVWT52XkRiZQ11T1c//EABURAQEAAAAAAAAAAAAAAAAAABEg/9oACAEDAQE/ASP/xAAWEQEBAQAAAAAAAAAAAAAAAAARABD/2gAIAQIBAT8BZwv/xAAcEAABBAMBAAAAAAAAAAAAAAAAEBEiMQEhMpH/2gAIAQEABj8C6ySoi/i0jnJpf//EABwQAQADAQADAQAAAAAAAAAAAAEAESExQVFhgf/aAAgBAQABPyHlWEAKvyh+AfY41s2z6hYst+S66BrYwpNe3HiJPdwLcxIk7Bbs/9oADAMBAAIAAwAAABDE5b3/xAAWEQEBAQAAAAAAAAAAAAAAAAAAEQH/2gAIAQMBAT8QykRX/8QAFxEBAQEBAAAAAAAAAAAAAAAAAQAQEf/aAAgBAgEBPxAV1GP/xAAfEAEAAgIDAAMBAAAAAAAAAAABESEAQTFRcWGBkaH/2gAIAQEAAT8QqUIBFf1i8Xg/BLPIxWtlEs/uEpR7vC1JXb1gMkcSSxnhrDYd3gONKwfXeJ99GzDgTZKfmMZKJN+cYdIgHXhn/9k=",
            "aspectRatio": 0.75,
            "src": "/static/5b11aa8932c0f2a7d09fdf4f745e4f90/d47f1/tim.jpg",
            "srcSet": "/static/5b11aa8932c0f2a7d09fdf4f745e4f90/b5b92/tim.jpg 200w,\n/static/5b11aa8932c0f2a7d09fdf4f745e4f90/3ff6e/tim.jpg 400w,\n/static/5b11aa8932c0f2a7d09fdf4f745e4f90/d47f1/tim.jpg 800w,\n/static/5b11aa8932c0f2a7d09fdf4f745e4f90/86173/tim.jpg 1200w,\n/static/5b11aa8932c0f2a7d09fdf4f745e4f90/a5547/tim.jpg 1600w,\n/static/5b11aa8932c0f2a7d09fdf4f745e4f90/40ffe/tim.jpg 2400w,\n/static/5b11aa8932c0f2a7d09fdf4f745e4f90/8a8e6/tim.jpg 3024w",
            "sizes": "(max-width: 800px) 100vw, 800px"
          }
        }
      }
    }
    
    const tree = renderer.create(<Index data={data} />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
