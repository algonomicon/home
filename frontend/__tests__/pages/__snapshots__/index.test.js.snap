// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Index renders correctly 1`] = `
<div>
  <div
    className="sc-gZMcBi fkYhnm"
  >
    <div
      className="sc-htoDjs exnspz"
    >
      <div
        className="sc-gqjmRU ctxItp"
      >
        <div
          className="logo"
          onMouseEnter={[Function]}
          onMouseLeave={[Function]}
        >
          <a
            className="glitch"
            href="/"
          >
            <svg
              className="glitch__img"
              height="31.75mm"
              version="1.1"
              viewBox="0 0 1170 112.5"
              width="330.2mm"
            >
              <g
                id="layer1"
                transform="translate(962.86 -165.58)"
              >
                <g
                  id="text3336"
                >
                  <path
                    d="m-940.36 188.08v-22.5h45v22.5h22.5v90h-22.5v-22.5h-45v22.5h-22.5v-90h22.5zm45 45v-44.82h-45v44.82h45z"
                    id="path3364"
                  />
                  <path
                    d="m-782.86 278.08v-22.5h-45v-90h-22.5v112.5h67.5z"
                    id="path3366"
                  />
                  <path
                    d="m-670.36 188.08v-22.5h-67.5v22.5h67.5zm-67.5 0h-22.5v67.5h22.5v-67.5zm67.5 90h-67.5v-22.5h45v-22.5h-22.5v-22.5h45v67.5z"
                    id="path3368"
                  />
                  <path
                    d="m-580.36 255.4v-67.14h-45v67.14h45zm-45 0.18h-22.5v-67.5h22.5v-22.5h45v22.5h22.5v67.5h-22.5v22.5h-45v-22.5z"
                    id="path3370"
                  />
                  <path
                    d="m-467.86 233.08v45h22.5v-112.5h-22.5v45h-22.5v22.5h22.5zm-45-22.5h22.5v-22.5h-22.5v-22.5h-22.5v112.5h22.5v-67.5z"
                    id="path3372"
                  />
                  <path
                    d="m-355.36 255.4v-67.14h-45v67.14h45zm-45 0.18h-22.5v-67.5h22.5v-22.5h45v22.5h22.5v67.5h-22.5v22.5h-45v-22.5z"
                    id="path3374"
                  />
                  <path
                    d="m-287.86 188.08v-22.5h-22.5v112.5h22.5v-67.5h22.5v-22.5h-22.5zm22.5 45h22.5v-22.5h-22.5v22.5zm45-45h-22.5v22.5h22.5v67.5h22.5v-112.5h-22.5v22.5z"
                    id="path3376"
                  />
                  <path
                    d="m-130.36 188.08h22.5v-22.5h-67.5v22.5h22.5v67.5h-22.5v22.5h67.5v-22.5h-22.5v-67.5z"
                    id="path3378"
                  />
                  <path
                    d="m-17.857 188.08v-22.5h-45v22.5h45zm-45 0h-22.5v67.5h22.5v-67.5zm45 90v-22.5h-45v22.5h45z"
                    id="path3380"
                  />
                  <path
                    d="m72.143 255.4v-67.14h-45v67.14h45zm-45 0.18h-22.5v-67.5h22.5v-22.5h45v22.5h22.5v67.5h-22.5v22.5h-45v-22.5z"
                    id="path3382"
                  />
                  <path
                    d="m184.64 233.08v45h22.5v-112.5h-22.5v45h-22.5v22.5h22.5zm-45-22.5h22.5v-22.5h-22.5v-22.5h-22.5v112.5h22.5v-67.5z"
                    id="path3384"
                  />
                </g>
              </g>
            </svg>
            <svg
              className="glitch__img"
              height="31.75mm"
              version="1.1"
              viewBox="0 0 1170 112.5"
              width="330.2mm"
            >
              <g
                id="layer1"
                transform="translate(962.86 -165.58)"
              >
                <g
                  id="text3336"
                >
                  <path
                    d="m-940.36 188.08v-22.5h45v22.5h22.5v90h-22.5v-22.5h-45v22.5h-22.5v-90h22.5zm45 45v-44.82h-45v44.82h45z"
                    id="path3364"
                  />
                  <path
                    d="m-782.86 278.08v-22.5h-45v-90h-22.5v112.5h67.5z"
                    id="path3366"
                  />
                  <path
                    d="m-670.36 188.08v-22.5h-67.5v22.5h67.5zm-67.5 0h-22.5v67.5h22.5v-67.5zm67.5 90h-67.5v-22.5h45v-22.5h-22.5v-22.5h45v67.5z"
                    id="path3368"
                  />
                  <path
                    d="m-580.36 255.4v-67.14h-45v67.14h45zm-45 0.18h-22.5v-67.5h22.5v-22.5h45v22.5h22.5v67.5h-22.5v22.5h-45v-22.5z"
                    id="path3370"
                  />
                  <path
                    d="m-467.86 233.08v45h22.5v-112.5h-22.5v45h-22.5v22.5h22.5zm-45-22.5h22.5v-22.5h-22.5v-22.5h-22.5v112.5h22.5v-67.5z"
                    id="path3372"
                  />
                  <path
                    d="m-355.36 255.4v-67.14h-45v67.14h45zm-45 0.18h-22.5v-67.5h22.5v-22.5h45v22.5h22.5v67.5h-22.5v22.5h-45v-22.5z"
                    id="path3374"
                  />
                  <path
                    d="m-287.86 188.08v-22.5h-22.5v112.5h22.5v-67.5h22.5v-22.5h-22.5zm22.5 45h22.5v-22.5h-22.5v22.5zm45-45h-22.5v22.5h22.5v67.5h22.5v-112.5h-22.5v22.5z"
                    id="path3376"
                  />
                  <path
                    d="m-130.36 188.08h22.5v-22.5h-67.5v22.5h22.5v67.5h-22.5v22.5h67.5v-22.5h-22.5v-67.5z"
                    id="path3378"
                  />
                  <path
                    d="m-17.857 188.08v-22.5h-45v22.5h45zm-45 0h-22.5v67.5h22.5v-67.5zm45 90v-22.5h-45v22.5h45z"
                    id="path3380"
                  />
                  <path
                    d="m72.143 255.4v-67.14h-45v67.14h45zm-45 0.18h-22.5v-67.5h22.5v-22.5h45v22.5h22.5v67.5h-22.5v22.5h-45v-22.5z"
                    id="path3382"
                  />
                  <path
                    d="m184.64 233.08v45h22.5v-112.5h-22.5v45h-22.5v22.5h22.5zm-45-22.5h22.5v-22.5h-22.5v-22.5h-22.5v112.5h22.5v-67.5z"
                    id="path3384"
                  />
                </g>
              </g>
            </svg>
            <svg
              className="glitch__img"
              height="31.75mm"
              version="1.1"
              viewBox="0 0 1170 112.5"
              width="330.2mm"
            >
              <g
                id="layer1"
                transform="translate(962.86 -165.58)"
              >
                <g
                  id="text3336"
                >
                  <path
                    d="m-940.36 188.08v-22.5h45v22.5h22.5v90h-22.5v-22.5h-45v22.5h-22.5v-90h22.5zm45 45v-44.82h-45v44.82h45z"
                    id="path3364"
                  />
                  <path
                    d="m-782.86 278.08v-22.5h-45v-90h-22.5v112.5h67.5z"
                    id="path3366"
                  />
                  <path
                    d="m-670.36 188.08v-22.5h-67.5v22.5h67.5zm-67.5 0h-22.5v67.5h22.5v-67.5zm67.5 90h-67.5v-22.5h45v-22.5h-22.5v-22.5h45v67.5z"
                    id="path3368"
                  />
                  <path
                    d="m-580.36 255.4v-67.14h-45v67.14h45zm-45 0.18h-22.5v-67.5h22.5v-22.5h45v22.5h22.5v67.5h-22.5v22.5h-45v-22.5z"
                    id="path3370"
                  />
                  <path
                    d="m-467.86 233.08v45h22.5v-112.5h-22.5v45h-22.5v22.5h22.5zm-45-22.5h22.5v-22.5h-22.5v-22.5h-22.5v112.5h22.5v-67.5z"
                    id="path3372"
                  />
                  <path
                    d="m-355.36 255.4v-67.14h-45v67.14h45zm-45 0.18h-22.5v-67.5h22.5v-22.5h45v22.5h22.5v67.5h-22.5v22.5h-45v-22.5z"
                    id="path3374"
                  />
                  <path
                    d="m-287.86 188.08v-22.5h-22.5v112.5h22.5v-67.5h22.5v-22.5h-22.5zm22.5 45h22.5v-22.5h-22.5v22.5zm45-45h-22.5v22.5h22.5v67.5h22.5v-112.5h-22.5v22.5z"
                    id="path3376"
                  />
                  <path
                    d="m-130.36 188.08h22.5v-22.5h-67.5v22.5h22.5v67.5h-22.5v22.5h67.5v-22.5h-22.5v-67.5z"
                    id="path3378"
                  />
                  <path
                    d="m-17.857 188.08v-22.5h-45v22.5h45zm-45 0h-22.5v67.5h22.5v-67.5zm45 90v-22.5h-45v22.5h45z"
                    id="path3380"
                  />
                  <path
                    d="m72.143 255.4v-67.14h-45v67.14h45zm-45 0.18h-22.5v-67.5h22.5v-22.5h45v22.5h22.5v67.5h-22.5v22.5h-45v-22.5z"
                    id="path3382"
                  />
                  <path
                    d="m184.64 233.08v45h22.5v-112.5h-22.5v45h-22.5v22.5h22.5zm-45-22.5h22.5v-22.5h-22.5v-22.5h-22.5v112.5h22.5v-67.5z"
                    id="path3384"
                  />
                </g>
              </g>
            </svg>
            <svg
              className="glitch__img"
              height="31.75mm"
              version="1.1"
              viewBox="0 0 1170 112.5"
              width="330.2mm"
            >
              <g
                id="layer1"
                transform="translate(962.86 -165.58)"
              >
                <g
                  id="text3336"
                >
                  <path
                    d="m-940.36 188.08v-22.5h45v22.5h22.5v90h-22.5v-22.5h-45v22.5h-22.5v-90h22.5zm45 45v-44.82h-45v44.82h45z"
                    id="path3364"
                  />
                  <path
                    d="m-782.86 278.08v-22.5h-45v-90h-22.5v112.5h67.5z"
                    id="path3366"
                  />
                  <path
                    d="m-670.36 188.08v-22.5h-67.5v22.5h67.5zm-67.5 0h-22.5v67.5h22.5v-67.5zm67.5 90h-67.5v-22.5h45v-22.5h-22.5v-22.5h45v67.5z"
                    id="path3368"
                  />
                  <path
                    d="m-580.36 255.4v-67.14h-45v67.14h45zm-45 0.18h-22.5v-67.5h22.5v-22.5h45v22.5h22.5v67.5h-22.5v22.5h-45v-22.5z"
                    id="path3370"
                  />
                  <path
                    d="m-467.86 233.08v45h22.5v-112.5h-22.5v45h-22.5v22.5h22.5zm-45-22.5h22.5v-22.5h-22.5v-22.5h-22.5v112.5h22.5v-67.5z"
                    id="path3372"
                  />
                  <path
                    d="m-355.36 255.4v-67.14h-45v67.14h45zm-45 0.18h-22.5v-67.5h22.5v-22.5h45v22.5h22.5v67.5h-22.5v22.5h-45v-22.5z"
                    id="path3374"
                  />
                  <path
                    d="m-287.86 188.08v-22.5h-22.5v112.5h22.5v-67.5h22.5v-22.5h-22.5zm22.5 45h22.5v-22.5h-22.5v22.5zm45-45h-22.5v22.5h22.5v67.5h22.5v-112.5h-22.5v22.5z"
                    id="path3376"
                  />
                  <path
                    d="m-130.36 188.08h22.5v-22.5h-67.5v22.5h22.5v67.5h-22.5v22.5h67.5v-22.5h-22.5v-67.5z"
                    id="path3378"
                  />
                  <path
                    d="m-17.857 188.08v-22.5h-45v22.5h45zm-45 0h-22.5v67.5h22.5v-67.5zm45 90v-22.5h-45v22.5h45z"
                    id="path3380"
                  />
                  <path
                    d="m72.143 255.4v-67.14h-45v67.14h45zm-45 0.18h-22.5v-67.5h22.5v-22.5h45v22.5h22.5v67.5h-22.5v22.5h-45v-22.5z"
                    id="path3382"
                  />
                  <path
                    d="m184.64 233.08v45h22.5v-112.5h-22.5v45h-22.5v22.5h22.5zm-45-22.5h22.5v-22.5h-22.5v-22.5h-22.5v112.5h22.5v-67.5z"
                    id="path3384"
                  />
                </g>
              </g>
            </svg>
            <svg
              className="glitch__img"
              height="31.75mm"
              version="1.1"
              viewBox="0 0 1170 112.5"
              width="330.2mm"
            >
              <g
                id="layer1"
                transform="translate(962.86 -165.58)"
              >
                <g
                  id="text3336"
                >
                  <path
                    d="m-940.36 188.08v-22.5h45v22.5h22.5v90h-22.5v-22.5h-45v22.5h-22.5v-90h22.5zm45 45v-44.82h-45v44.82h45z"
                    id="path3364"
                  />
                  <path
                    d="m-782.86 278.08v-22.5h-45v-90h-22.5v112.5h67.5z"
                    id="path3366"
                  />
                  <path
                    d="m-670.36 188.08v-22.5h-67.5v22.5h67.5zm-67.5 0h-22.5v67.5h22.5v-67.5zm67.5 90h-67.5v-22.5h45v-22.5h-22.5v-22.5h45v67.5z"
                    id="path3368"
                  />
                  <path
                    d="m-580.36 255.4v-67.14h-45v67.14h45zm-45 0.18h-22.5v-67.5h22.5v-22.5h45v22.5h22.5v67.5h-22.5v22.5h-45v-22.5z"
                    id="path3370"
                  />
                  <path
                    d="m-467.86 233.08v45h22.5v-112.5h-22.5v45h-22.5v22.5h22.5zm-45-22.5h22.5v-22.5h-22.5v-22.5h-22.5v112.5h22.5v-67.5z"
                    id="path3372"
                  />
                  <path
                    d="m-355.36 255.4v-67.14h-45v67.14h45zm-45 0.18h-22.5v-67.5h22.5v-22.5h45v22.5h22.5v67.5h-22.5v22.5h-45v-22.5z"
                    id="path3374"
                  />
                  <path
                    d="m-287.86 188.08v-22.5h-22.5v112.5h22.5v-67.5h22.5v-22.5h-22.5zm22.5 45h22.5v-22.5h-22.5v22.5zm45-45h-22.5v22.5h22.5v67.5h22.5v-112.5h-22.5v22.5z"
                    id="path3376"
                  />
                  <path
                    d="m-130.36 188.08h22.5v-22.5h-67.5v22.5h22.5v67.5h-22.5v22.5h67.5v-22.5h-22.5v-67.5z"
                    id="path3378"
                  />
                  <path
                    d="m-17.857 188.08v-22.5h-45v22.5h45zm-45 0h-22.5v67.5h22.5v-67.5zm45 90v-22.5h-45v22.5h45z"
                    id="path3380"
                  />
                  <path
                    d="m72.143 255.4v-67.14h-45v67.14h45zm-45 0.18h-22.5v-67.5h22.5v-22.5h45v22.5h22.5v67.5h-22.5v22.5h-45v-22.5z"
                    id="path3382"
                  />
                  <path
                    d="m184.64 233.08v45h22.5v-112.5h-22.5v45h-22.5v22.5h22.5zm-45-22.5h22.5v-22.5h-22.5v-22.5h-22.5v112.5h22.5v-67.5z"
                    id="path3384"
                  />
                </g>
              </g>
            </svg>
          </a>
        </div>
        <div
          className="sc-VigVT cQowiX"
        >
          <a
            className="sc-jTzLTM XXXSU"
            href="/about"
          >
            About
          </a>
          <a
            className="sc-jTzLTM XXXSU"
            href="/contact"
          >
            Contact
          </a>
        </div>
      </div>
      <div
        className="sc-fjdhpX jVgRYA"
      >
        <h1
          className="sc-jzJRlG diYmFi"
        >
          Machine Learning for Fun and Profit
        </h1>
        <h4
          className="sc-cSHVUG dHvNZW"
        >
          Still under construction. Thanks for bearing with us! Sign up for updates.
        </h4>
        <div
          className="sc-kAzzGY iGneOL"
        >
          <div
            className="sc-kgoBCf heGha"
          >
            <div
              className="sc-kGXeez lmfQek"
            >
              <svg
                aria-hidden="true"
                className="svg-inline--fa fa-user-astronaut fa-w-14 fa-3x"
                data-icon="user-astronaut"
                data-prefix="fas"
                role="img"
                style={
                  Object {
                    "display": "block",
                  }
                }
                viewBox="0 0 448 512"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M64 224h13.5c24.7 56.5 80.9 96 146.5 96s121.8-39.5 146.5-96H384c8.8 0 16-7.2 16-16v-96c0-8.8-7.2-16-16-16h-13.5C345.8 39.5 289.6 0 224 0S102.2 39.5 77.5 96H64c-8.8 0-16 7.2-16 16v96c0 8.8 7.2 16 16 16zm40-88c0-22.1 21.5-40 48-40h144c26.5 0 48 17.9 48 40v24c0 53-43 96-96 96h-48c-53 0-96-43-96-96v-24zm72 72l12-36 36-12-36-12-12-36-12 36-36 12 36 12 12 36zm151.6 113.4C297.7 340.7 262.2 352 224 352s-73.7-11.3-103.6-30.6C52.9 328.5 0 385 0 454.4v9.6c0 26.5 21.5 48 48 48h80v-64c0-17.7 14.3-32 32-32h128c17.7 0 32 14.3 32 32v64h80c26.5 0 48-21.5 48-48v-9.6c0-69.4-52.9-125.9-120.4-133zM272 448c-8.8 0-16 7.2-16 16s7.2 16 16 16 16-7.2 16-16-7.2-16-16-16zm-96 0c-8.8 0-16 7.2-16 16v48h32v-48c0-8.8-7.2-16-16-16z"
                  fill="currentColor"
                />
              </svg>
            </div>
            <input
              className="sc-kpOJdX botntV"
              placeholder="example@algonomicon.com"
              type="text"
            />
          </div>
          <button
            className="sc-chPdSV hEdLZH"
          >
            Sign Up!
          </button>
        </div>
      </div>
    </div>
  </div>
  <div
    className="sc-dxgOiQ bGYNPR"
  >
    <div
      className="sc-htoDjs exnspz"
    >
      <div
        className="sc-bdVaJa knWkmq"
      >
        <a
          className="sc-bwzfXH FtMCK"
          href="/"
        >
          Home
        </a>
        <a
          className="sc-bwzfXH FtMCK"
          href="/articles"
          partiallyActive={true}
        >
          Articles
        </a>
        <a
          className="sc-bwzfXH FtMCK"
          href="/datasets"
          partiallyActive={true}
        >
          Datasets
        </a>
        <a
          className="sc-bwzfXH FtMCK"
          href="/papers"
          partiallyActive={true}
        >
          Papers
        </a>
        <a
          className="sc-bwzfXH FtMCK"
          href="/projects"
          partiallyActive={true}
        >
          Projects
        </a>
        <a
          className="sc-bwzfXH FtMCK"
          href="/snippets"
          partiallyActive={true}
        >
          Snippets
        </a>
      </div>
      <div
        className="sc-ckVGcZ jecudI"
      >
        <div
          className="sc-jKJlTe xPVez"
        >
          <p>
            Thanks to increased power and large amounts of data, machine learning models are capable of solving problems previously thought to be impossible. Machine Learning applications are driving cars, improving healthcare diagnosis, and sequencing new proteins. Machine learning has the power to do a lot of good and we want to tell the stories of people using it to change the world.
          </p>
          <div
            className=" gatsby-image-wrapper"
            style={
              Object {
                "overflow": "hidden",
                "position": "relative",
              }
            }
          >
            <div
              style={
                Object {
                  "paddingBottom": "133.33333333333334%",
                  "width": "100%",
                }
              }
            />
            <img
              alt=""
              src="data:image/jpeg;base64,/9j/2wBDABALDA4MChAODQ4SERATGCgaGBYWGDEjJR0oOjM9PDkzODdASFxOQERXRTc4UG1RV19iZ2hnPk1xeXBkeFxlZ2P/2wBDARESEhgVGC8aGi9jQjhCY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2P/wgARCAAbABQDASIAAhEBAxEB/8QAGAAAAwEBAAAAAAAAAAAAAAAAAAMEAgX/xAAWAQEBAQAAAAAAAAAAAAAAAAABAAL/2gAMAwEAAhADEAAAAVtgC6pyyqMUKNYGif/EAB4QAAIBBAMBAAAAAAAAAAAAAAABAgQRITEDEhNC/9oACAEBAAEFAvfkIVLUpVWT52XkRiZQ11T1c//EABURAQEAAAAAAAAAAAAAAAAAABEg/9oACAEDAQE/ASP/xAAWEQEBAQAAAAAAAAAAAAAAAAARABD/2gAIAQIBAT8BZwv/xAAcEAABBAMBAAAAAAAAAAAAAAAAEBEiMQEhMpH/2gAIAQEABj8C6ySoi/i0jnJpf//EABwQAQADAQADAQAAAAAAAAAAAAEAESExQVFhgf/aAAgBAQABPyHlWEAKvyh+AfY41s2z6hYst+S66BrYwpNe3HiJPdwLcxIk7Bbs/9oADAMBAAIAAwAAABDE5b3/xAAWEQEBAQAAAAAAAAAAAAAAAAAAEQH/2gAIAQMBAT8QykRX/8QAFxEBAQEBAAAAAAAAAAAAAAAAAQAQEf/aAAgBAgEBPxAV1GP/xAAfEAEAAgIDAAMBAAAAAAAAAAABESEAQTFRcWGBkaH/2gAIAQEAAT8QqUIBFf1i8Xg/BLPIxWtlEs/uEpR7vC1JXb1gMkcSSxnhrDYd3gONKwfXeJ99GzDgTZKfmMZKJN+cYdIgHXhn/9k="
              style={
                Object {
                  "height": "100%",
                  "left": 0,
                  "objectFit": "cover",
                  "objectPosition": "center",
                  "opacity": 1,
                  "position": "absolute",
                  "top": 0,
                  "transitionDelay": "0.5s",
                  "width": "100%",
                }
              }
            />
            <picture>
              <img
                alt="Inception"
                onLoad={[Function]}
                sizes="(max-width: 800px) 100vw, 800px"
                src="/static/5b11aa8932c0f2a7d09fdf4f745e4f90/d47f1/tim.jpg"
                srcSet="/static/5b11aa8932c0f2a7d09fdf4f745e4f90/b5b92/tim.jpg 200w,
/static/5b11aa8932c0f2a7d09fdf4f745e4f90/3ff6e/tim.jpg 400w,
/static/5b11aa8932c0f2a7d09fdf4f745e4f90/d47f1/tim.jpg 800w,
/static/5b11aa8932c0f2a7d09fdf4f745e4f90/86173/tim.jpg 1200w,
/static/5b11aa8932c0f2a7d09fdf4f745e4f90/a5547/tim.jpg 1600w,
/static/5b11aa8932c0f2a7d09fdf4f745e4f90/40ffe/tim.jpg 2400w,
/static/5b11aa8932c0f2a7d09fdf4f745e4f90/8a8e6/tim.jpg 3024w"
                style={
                  Object {
                    "height": "100%",
                    "left": 0,
                    "objectFit": "cover",
                    "objectPosition": "center",
                    "opacity": 0,
                    "position": "absolute",
                    "top": 0,
                    "transition": "opacity 0.5s",
                    "width": "100%",
                  }
                }
              />
            </picture>
            <noscript
              dangerouslySetInnerHTML={
                Object {
                  "__html": "<picture><img sizes=\\"(max-width: 800px) 100vw, 800px\\" srcset=\\"/static/5b11aa8932c0f2a7d09fdf4f745e4f90/b5b92/tim.jpg 200w,
/static/5b11aa8932c0f2a7d09fdf4f745e4f90/3ff6e/tim.jpg 400w,
/static/5b11aa8932c0f2a7d09fdf4f745e4f90/d47f1/tim.jpg 800w,
/static/5b11aa8932c0f2a7d09fdf4f745e4f90/86173/tim.jpg 1200w,
/static/5b11aa8932c0f2a7d09fdf4f745e4f90/a5547/tim.jpg 1600w,
/static/5b11aa8932c0f2a7d09fdf4f745e4f90/40ffe/tim.jpg 2400w,
/static/5b11aa8932c0f2a7d09fdf4f745e4f90/8a8e6/tim.jpg 3024w\\" src=\\"/static/5b11aa8932c0f2a7d09fdf4f745e4f90/d47f1/tim.jpg\\" alt=\\"Inception\\" style=\\"position:absolute;top:0;left:0;opacity:1;width:100%;height:100%;object-fit:cover;object-position:center\\"/></picture>",
                }
              }
            />
          </div>
          <p
            style={
              Object {
                "margin": "1rem 0",
              }
            }
          >
            <i>
              Inception: A state of the art deep learning architecture developed by the AI research team at Google.
            </i>
          </p>
          <blockquote>
            Truth is much too complicated to allow anything but approximations.
            <p>
              <i>
                - John Von Neumann
              </i>
            </p>
          </blockquote>
          <p>
            We're particularly interested in the following research topics:
          </p>
          <ul>
            <li>
              Climate Change and Anthropogenic Impacts on the Environment
            </li>
            <li>
              Water Scarcity and Pollution
            </li>
            <li>
              Poverty, Disease and Malnutrition
            </li>
            <li>
              Overpopulation and Global Education
            </li>
            <li>
              Cyber Threats and Regional Stability
            </li>
            <li>
              Wildlife Conservation and Biodiversity Loss
            </li>
            <li>
              Renewable and Alternative Energy
            </li>
          </ul>
          <p>
            If you own or work at a company that is tackling global or social issues, we'd love to get in touch. Feel free to visit our
             
            <a
              href="/contact"
            >
              contact page
            </a>
             or reach out directly to info@algonomicon.com.
          </p>
          <p>
            We support open source software. If you like our content, please join our mission and collaborate with us on
             
            <a
              href="https://github.com/algonomicon"
            >
              Github
            </a>
            .
          </p>
        </div>
        <div
          className="sc-cMljjf jdrAFH"
        >
          <div>
            <h6
              className="sc-htpNat iacXlz"
            >
              Feed
            </h6>
            <div
              className="sc-bxivhb kphxJa"
            >
              <div>
                <a
                  className="sc-ifAKCX gYXmWr"
                  href="/snippets/hello-world"
                >
                  <h4
                    className="sc-EHOje eYGxhL"
                  >
                    Hello World
                  </h4>
                  <p
                    className="sc-bZQynM hDSPrr"
                  >
                    Lorem ipsum dolor sit amet.
                    ...
                  </p>
                  <small
                    className="sc-gzVnrw bpUwpJ"
                  >
                    Snippet from 
                    6 days ago
                  </small>
                </a>
              </div>
              <div>
                <a
                  className="sc-ifAKCX gYXmWr"
                  href="/articles/getting-flux-and-cuda-to-work-with-julia-pro"
                >
                  <h4
                    className="sc-EHOje eYGxhL"
                  >
                    Getting Flux and Cuda to Work with Julia Pro
                  </h4>
                  <p
                    className="sc-bZQynM hDSPrr"
                  >
                    The fundamental concepts of classical physics, space, time, mass, and derived concepts, velocity, mo
                    ...
                  </p>
                  <small
                    className="sc-gzVnrw bpUwpJ"
                  >
                    Article from 
                    9 days ago
                  </small>
                </a>
              </div>
              <div>
                <a
                  className="sc-ifAKCX gYXmWr"
                  href="/projects/coming-soon"
                >
                  <h4
                    className="sc-EHOje eYGxhL"
                  >
                    Coming Soon
                  </h4>
                  <p
                    className="sc-bZQynM hDSPrr"
                  >
                    Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approache
                    ...
                  </p>
                  <small
                    className="sc-gzVnrw bpUwpJ"
                  >
                    Project from 
                    22 days ago
                  </small>
                </a>
              </div>
              <div>
                <a
                  className="sc-ifAKCX gYXmWr"
                  href="/papers/a-neural-algorithm-of-artistic-style"
                >
                  <h4
                    className="sc-EHOje eYGxhL"
                  >
                    A Neural Algorithm of Artistic Style
                  </h4>
                  <p
                    className="sc-bZQynM hDSPrr"
                  >
                    In fine art, especially painting, humans have mastered the skill to create unique visual experiences
                    ...
                  </p>
                  <small
                    className="sc-gzVnrw bpUwpJ"
                  >
                    Paper from 
                    22 days ago
                  </small>
                </a>
              </div>
              <div>
                <a
                  className="sc-ifAKCX gYXmWr"
                  href="/datasets/mnist"
                >
                  <h4
                    className="sc-EHOje eYGxhL"
                  >
                    MNIST
                  </h4>
                  <p
                    className="sc-bZQynM hDSPrr"
                  >
                    Handwritten digits
                    ...
                  </p>
                  <small
                    className="sc-gzVnrw bpUwpJ"
                  >
                    Dataset from 
                    23 days ago
                  </small>
                </a>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
  <div
    className="sc-dnqmqq ipcvRL"
  >
    <div
      className="sc-htoDjs exnspz"
    >
      <p
        className="sc-iwsKbI bYpdxf"
      >
        © Algonomicon LLC 2019
      </p>
    </div>
  </div>
</div>
`;
